chosww/idrc
# feat: implement front-end style for translation links (resolves inclusive-design/idrc#629)

## Description

Implements @cherylhjli's icon and link style for the translation links.

## Steps to test

1. Add a French translation for a project.
2. Visit the project page.

**Expected behavior:** Style matches [designs](https://www.figma.com/file/GyPtCZtqRWxpoKLPEMn05d/IDRCWebsite_Screens_2022---Projects-pages-revision?type=design&node-id=497%3A3147&t=Fi6NhLtoQwsO6BNy-1)[https://www.figma.com/file/GyPtCZtqRWxpoKLPEMn05d/IDRCWebsite_Screens_2022---Projects-pages-revision?type=design&node-id=497%3A3147&t=Fi6NhLtoQwsO6BNy-1](https://can01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.figma.com%2Ffile%2FGyPtCZtqRWxpoKLPEMn05d%2FIDRCWebsite_Screens_2022---Projects-pages-revision%3Ftype%3Ddesign%26node-id%3D497%253A3147%26t%3DFi6NhLtoQwsO6BNy-1&data=05%7C01%7Cnzimmerman%40ocadu.ca%7C035c687e67bf4a830bf608db7120ad78%7C06e469d12d2a468fae9b7df0968eb6d7%7C0%7C0%7C638228157384228338%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=8mMsuwJ01xlYxwRbgFkVxzZzCAY1L0I7G687fRXXoKc%3D&reserved=0).

## Additional information

The icon was a bit distorted in the Figma file so I fixed it.

## Related issues

Resolves inclusive-design/idrc#629.
